tests/datasource/data_connector/test_inferred_asset_filesystem_data_connector.py

```diff
@@ -292,193 +291,6 @@ def test_self_check(tmp_path_factory):
     }
 
 
-@mock.patch(
-    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
-)
-def test_test_yaml_config(
-    mock_emit, empty_data_context_stats_enabled, tmp_path_factory
-):
-    context = empty_data_context_stats_enabled
-
-    base_directory = str(tmp_path_factory.mktemp("test_test_yaml_config"))
-    create_files_in_directory(
-        directory=base_directory,
-        file_name_list=[
-            "2020/01/alpha-1001.csv",
-            "2020/01/beta-1002.csv",
-            "2020/02/alpha-1003.csv",
-            "2020/02/beta-1004.csv",
-            "2020/03/alpha-1005.csv",
-            "2020/03/beta-1006.csv",
-            "2020/04/beta-1007.csv",
-        ],
-    )
-
-    report_object = context.test_yaml_config(
-        f"""
-module_name: great_expectations.datasource.data_connector
-class_name: InferredAssetFilesystemDataConnector
-datasource_name: FAKE_DATASOURCE
-name: TEST_DATA_CONNECTOR
-base_directory: {base_directory}/
-glob_directive: "*/*/*.csv"
-default_regex:
-    pattern: (\\d{{4}})/(\\d{{2}})/(.*)-.*\\.csv
-    group_names:
-        - year_dir
-        - month_dir
-        - data_asset_name
-    """,
-        runtime_environment={
-            "execution_engine": PandasExecutionEngine(),
-        },
-        return_mode="report_object",
-    )
-
-    assert report_object == {
```

Comment:
Maybe this is me lacking context. Should this file really be gone? Isn't testing the output from `test_yaml_config` still relevant, or is that going away?

Reply:
Ah yeah I think I deleted a few instances where we were using `test_yaml_config` in general since its no longer relevant - I can revert these changes